components/alertbanner/metadata/alertbanner.yml

```diff
@@ -1,5 +1,9 @@
-name: Alert banner
+name: Alert Banner
```

Comment:
So I believe the content direction is to use sentence case and not title case for content on the site so Alert banner would be correct over Alert Banner.  I also suggested the link to the mods be left off since it was going to be accounted for in the 11ty rework which is impending (just another diff I have to resolve)

Reply:
aha! Ok.